app/modules/individuals/models.py

```diff
@@ -91,6 +91,8 @@ class Individual(db.Model, FeatherModel):
         order_by='SocialGroupIndividualMembership.individual_guid',
     )
 
+    # there is a backref'd 'relationships' list of RelationshipIndividualMember accessible here
```

Comment:
In which case I find it easier to read if the thing exists in the module

Reply:
Back_populates wasn't working correctly, it's what I used originally. Switching to backref which does not require the thing to be in the module worked, so I added this note.